user/wlr-protocols/template.py

```diff
@@ -0,0 +1,19 @@
+pkgname = "wlr-protocols"
```

Comment:
don't really like this, i've never seen anything directly use this and it's yet another thing with no releases that nothing really uses

Reply:
you can add the one xml file drawterm needs into its sources and then pass `WLR_VIRTUAL_POINTER` on make commandline instead

Reply:
wlr-protocols is not meant to be packaged